t/03-proxy_wasm/sdks/002-go_sdk/005-dispatch_call_on_tick.t

```diff
@@ -46,15 +45,19 @@ Connection refused on 127.0.0.1:80 requested by filter.
         internal;
         # the filter does not support HTTP contexts
         proxy_wasm go_dispatch_call_on_tick;
```

Comment:
I currently don't have access to a TinyGo installation since it is out of sync with my Go installation (`error: requires go version 1.18 through 1.20, got go1.21`). So why this doesn't not pass, I don't know and unfortunately cannot investigate.

Reply:
Nice, since we upload failed jobs's error logs to GHA I got to the root cause of it.